### PR TITLE
Feat : Notes Container On Live

### PR DESCRIPTION
**Pull request by Rishi05051997**
Peer members, please review the notes container code
notes-container

### CODE REVIEW

File: src/components/Notes-Container/NoteCard.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import { deleteNoteById } from "../../services";
+import { useData } from "../../context/DataProvider";
+import { ColorContainer } from "../ColorPallet/ColorContainer";
+import { Selectbox } from "../SelectBox/Selectbox";
+import { ArchiveNoteById } from "../../services";
+import { useNavigate } from "react-router-dom";
+import { AddNote } from "../NoteForm/AddNote";
+import { useState } from "react";
+// import { deleteLableHandler } from "../../services";
+// import Select from 'react-select';
```
Comment (Mohasin-Haque) [inline src/components/Notes-Container/NoteCard.jsx]: please remove the commented code.

File: src/components/Notes-Container/NoteCard.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                        < ColorContainer noteId={_id} note={note} />
+                        < Selectbox options={priority} dispatch={dispatch} type="PRIORITY-UPDATE" noteId={_id} note={note} />
+                        <Icon className="iconify cursor_" icon="bxs:archive-in" onClick={() => ArchiveNoteById(dispatch, _id, note, token, navigate)} />
+                        <Icon className="iconify cursor_" icon="akar-icons:edit" onClick={() => setIsEditable(val => !val)} />
+                        {/* <Select
+                        closeMenuOnSelect={false}
+                        isMulti
+                        options={selectedLabels}
+                        onChange={(e) => ({ ...note, selectedLabels: [...note.selectedLabels].concat(e) })}
+                    /> */}
```
Comment (Mohasin-Haque) [inline src/components/Notes-Container/NoteCard.jsx]: same here also

**Review by Mohasin-Haque — APPROVED** [review]
You can work on sanity , rest is good

File: src/components/Notes-Container/NoteCard.jsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                        </>
+                    }
+                </div>
+                <footer>
+                    <div className="card-footer text-5 mar-xs pad-xs">
+                        <Icon className="iconify cursor_" icon="fluent:delete-28-regular" onClick={() => deleteNoteById(dispatch, token, _id)} />
+                        < ColorContainer noteId={_id} note={note} />
+                        < Selectbox options={priority} dispatch={dispatch} type="PRIORITY-UPDATE" noteId={_id} note={note} />
+                        <Icon className="iconify cursor_" icon="bxs:archive-in" onClick={() => ArchiveNoteById(dispatch, _id, note, token, navigate)} />
+                        <Icon className="iconify cursor_" icon="akar-icons:edit" onClick={() => setIsEditable(val => !val)} />
```
Comment (vishalg8454) [inline src/components/Notes-Container/NoteCard.jsx:59]: why use of `footer` tag in note component?

**Review by vishalg8454 — APPROVED** [review]
Looks good to merge. Also remove the commented code as pointed by Mohasin too.

File: src/components/Notes-Container/NoteCard.jsx (hunk 3 above)
Comment (Rishi05051997) [reply src/components/Notes-Container/NoteCard.jsx:59]: As I'm trying to use Semantic Structure more that's why I used the footer over there like cards footer.

**Pull request merged**